back-end/test/us-03-create-reservations-eligible-timeframe.test.js

```diff
@@ -50,7 +50,7 @@ describe("US-03 - Create reservations eligible timeframe", () => {
         .send({ data });
       expect(response.status).toBe(400);
 
-      data.reservation_time = "15:30";
+      data.reservation_time = "05:30";
```

Comment:
Why was it necessary to change the time?

Reply:
the restaurant is open at 15:30. It should either be a different hour or return a 200.